src/components/Newsletter/Newsletter.js

```diff
@@ -1,12 +1,14 @@
 import firebase from 'firebase';
 import React from 'react';
-import { StyledNewsletter, InputHolder } from './Newsletter.style';
+import { StyledNewsletter, InputHolder, InputError } from './Newsletter.style';
 import Chip from '@material-ui/core/Chip';
 import Autocomplete from '@material-ui/lab/Autocomplete';
 import TextField from '@material-ui/core/TextField';
 import { getLocationNames } from 'common/locations';
 import { getFirebase } from 'common/firebase';
 
+const EMAIL_REGEX = /^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/;
+
```

Comment:
prettier (I guess?) wants you to unescape a bunch of things it seems.

Reply:
coook okay fixed